utilities/rpp-unittests/HIP_NEW/BatchPD_hip_pln1.cpp

```diff
@@ -765,6 +765,8 @@ int main(int argc, char **argv)
         else
             missingFuncFlag = 1;
 
+        hipDeviceSynchronize();
```

Comment:
Please move it outside the switch statement so you need to only make one line code change. Applies to all the files in this commit

Reply:
@rrawther The hipDeviceSynchronize() actually needs to be within the clock() timed section (before end = clock(); is called). Only then does the clock() wait for the GPU-offloads to finish entirely, and the end time can be recorded.

Reply:
https://github.com/r-abishek/rpp/blob/ar/test_suite_mods/utilities/rpp-unittests/HIP_NEW/Tensor_hip_pkd3.cpp#L577-L598

Reply:
probably you can move both outside the switch statement in that case. It is better to keep common code all at one place

Reply:
Ive now changed all instances in all test suite files to reflect this change. The end = clock() for HOST and the hipDeviceSynchronize() + end = clock() for HIP have been moved outside the switch case